src/backend/access/external/test/pxffilters_test.c

```diff
@@ -323,10 +323,32 @@ OpExpr* build_op_expr(void* left, void* right, int op)
 	return expr;
 }
 
-ExpressionItem* build_expression_item(int lattnum, Oid lattrtype, char* rconststr, Oid rattrtype, int op) {
+NullTest* build_null_expr(void* arg, NullTestType nullType)
```

Comment:
I just wanted the signature to be alligned with build_op_expr that also uses void* for the arguments. I will anyway update it to Expr * to avoid the extra casting